docs/features.md

```diff
@@ -0,0 +1,38 @@
+---
+Order: 20
+Title: Features
+Description: Features of the Cake.Issues.InspectCode addin.
+---
+The [Cake.Issues.InspectCode addin] provides the following features:
+
+# Basic features
+
+* Reads warnings from [JetBrains InsepectCode] log files.
+* Provides URLs for issues containing a Wiki URL.
+
+# Supported comment formats
+
+|                                                                    | Comment format                 | Remarks                               |
+|--------------------------------------------------------------------|--------------------------------|---------------------------------------|
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IssueCommentFormat.PlainText` |                                       |
+| <span class="glyphicon glyphicon-remove" style="color:red"></span> | `IssueCommentFormat.Markdown`  |                                       |
+| <span class="glyphicon glyphicon-remove" style="color:red"></span> | `IssueCommentFormat.Html`      |                                       |
+
+# Supported IIssue properties
```

Comment:
[MD025](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md025---multiple-top-level-headings-in-the-same-document
) multiple top level headings in the same document.